execution_workflows/DaPars2/Dapars2_snakemake/workflow/Snakefile

```diff
@@ -0,0 +1,333 @@
+"""Snakemake pipeline for Dapars2.
+For help see: https://snakemake.readthedocs.io/en/stable/index.html.
+"""
+
+import pandas as pd
+import os
+import sys
+
+configfile:"config/config.DaPars2.yaml"
+
+samples = pd.read_csv(config["samples"]).set_index("sample", drop=False)
+
+
+#-------------------------------------------------------------------------------
+localrules: finish
```

Comment:
Could it be that currently all rules are run locally and cluster execution is not set up yet? Maybe you could put a note somewhere, e.g. in README